server/auth/util.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2020 TiKV Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissionKeys and
+// limitations under the License.
+//
+
+package auth
+
+import (
+	"regexp"
+)
+
+var (
+	patName = regexp.MustCompile("^([A-Za-z])[A-Za-z0-9_]*$")
+)
```

Comment:
We can init `patName` in `init` func(), if any error happened, use log.Fatal to print the error and exit.

Reply:
Maybe this won't make any difference? If this regex fails, it will panic anyway, which is very unlikely to happen, and won't pass the unit test.

Reply:
That's ok to me.